src/app/shared/object-list/my-dspace-result-list-element/pool-search-result/pool-search-result-list-element.component.html

```diff
@@ -4,6 +4,15 @@
                                [showSubmitter]="showSubmitter"
                                [badgeContext]="badgeContext"
                                [workflowItem]="workflowitem$.value"></ds-themed-item-list-preview>
+  <!-- Display duplicate alert, if feature enabled and duplicates detected -->
```

Comment:
<img width="799" alt="Screenshot 2024-02-01 at 16 35 54" src="https://github.com/DSpace/dspace-angular/assets/8408850/731597d2-91dc-4a85-bdfc-ee7555f3e6c9">

Duplicate alert needs to be aligned with other lines.

Reply:
Fixed with a 'dummy' col to simulate thumbnail. 
![screenshot_2024-02-11_12:18:45_selection](https://github.com/DSpace/dspace-angular/assets/194773/6e25e16a-2c1e-4beb-85f6-fd7e640f159b)

Reply:
@kshepherd 

as suggested in [my comment](https://github.com/DSpace/dspace-angular/pull/2749#pullrequestreview-1883632104) for the submission also here i'd highlight the message with a warning alert or a text warning class

Reply:
done, thanks